worker/src/main/resources/alter_history_tables.sql

```diff
@@ -0,0 +1,11 @@
+START TRANSACTION;
+
+ALTER TABLE `job_history`
+  MODIFY COLUMN trigger_name VARCHAR(190) NOT NULL,
```

Comment:
@marwtki  Yes, but you might consider adding `job_name` and `job_group` here instead of replacing `trigger_name` and `trigger_group`.